src/ingest_llm_as/models/knowledge_digest.py

```
@@ -0,0 +1,70 @@
+"""Knowledge Digest model for OmegaKG Validation API.
+
+Defines the structure of the data sent from InGest-LLM workers to OmegaKG.
+"""
+from pydantic import BaseModel, Field
+from typing import List, Dict, Any, Optional
```

Ruff (0.14.11)

`typing.List` is deprecated, use `list` instead

(UP035)